applications/serial_lte_modem/doc/slm_description.rst

```diff
@@ -397,31 +377,47 @@ To connect with an external MCU using UART_2, change the configuration files for
 
 * In the :file:`nrf9160dk_nrf9160_ns.conf` file::
 
-   # Use UART_0 (when working with PC terminal)
-   # unmask the following config
-   #CONFIG_SLM_CONNECT_UART_0=y
-   #CONFIG_UART_0_NRF_HW_ASYNC_TIMER=2
-   #CONFIG_UART_0_NRF_HW_ASYNC=y
-   #CONFIG_SLM_WAKEUP_PIN=6
-   #CONFIG_SLM_INDICATE_PIN=2
+  .. code-block:: console
+
+     # Use UART_0 (when working with PC terminal)
+     # unmask the following config
+     #CONFIG_UART_0_NRF_HW_ASYNC_TIMER=2
+     #CONFIG_UART_0_NRF_HW_ASYNC=y
+     #CONFIG_SLM_WAKEUP_PIN=6
+     #CONFIG_SLM_INDICATE_PIN=2
 
-   # Use UART_2 (when working with external MCU)
-   # unmask the following config
-   CONFIG_SLM_CONNECT_UART_2=y
-   CONFIG_UART_2_NRF_HW_ASYNC_TIMER=2
-   CONFIG_UART_2_NRF_HW_ASYNC=y
-   CONFIG_SLM_WAKEUP_PIN=31
-   CONFIG_SLM_INDICATE_PIN=30
+     # Use UART_2 (when working with external MCU)
+     # unmask the following config
+     CONFIG_UART_2_NRF_HW_ASYNC_TIMER=2
+     CONFIG_UART_2_NRF_HW_ASYNC=y
+     CONFIG_SLM_WAKEUP_PIN=31
+     CONFIG_SLM_INDICATE_PIN=30
 
 
 * In the :file:`nrf9160dk_nrf9160_ns.overlay` file::
 
-   &uart0 {
+  .. code-block:: devicetree
+
+     / {
+	chosen {
+		ncs,slm-uart = &uart2;
+	       }
```

Comment:
@wiba-nordic please help with doc build failure with this line
/home/runner/work/sdk-nrf/sdk-nrf/ncs/nrf/doc/_build/nrf/src/applications/serial_lte_modem/doc/slm_description.rst:404:Block quote ends without a blank line; unexpected unindent.

Reply:
Not 100% sure, but it is likely tabs instead of spaces breaking the build. Line 402 should have 5 spaces so it lines with the c in code-block, and then no line should have less spaces. Since tab is generally seen as 4 spaces, it is misaligned and the system sees that as an unexpected unindent.

Try replacing all the tabs with spaces.